course_catalog/migrations/0077_course_department.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.13 on 2020-09-03 18:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("course_catalog", "0076_podcast_rss_url")]
+
+    operations = [
+        migrations.AddField(
+            model_name="course",
+            name="department",
+            field=models.CharField(blank=True, max_length=256, null=True),
+        )
+    ]
```

Comment:
Instead of running `backpopulate_ocw_data` which takes a really long time, might be faster to backfill from the course runs's raw_json field as part of the migration (assuming all OCW runs have populated raw_json fields).  Either way works though.

```python
def backfill_department(apps, schema_editor):
    """
    Backfills department values for existing Courses
    """
    Course = apps.get_model("course_catalog", "Course")
    for course in Course.objects.filter(
        platform=PlatformType.ocw.value
    ).iterator():
        course.department = course.runs.first().raw_json.get("department_number")
        course.save()
```